_posts/en/newsletters/2020-02-26-newsletter.md

```diff
@@ -0,0 +1,138 @@
+  described in [Newsletter #72][news72 offers]).  Russell's proposal
+  originally specified using the same there-and-back onion routing used
+  for LN payments (HTLCs), but developer ZmnSCPxj proposed having the
+  message sender specify the full path from their node to the message
+  recipient and back to the sender.  For example, if Alice wants to
+  communicate with Carol, she might choose the following path:
+
+        Alice → Bob → Carol → Dan → Alice
+
+    This type of circle routing would make surveillance more difficult
+    and would eliminate the overhead for routing nodes of having to
+    store the return path, making the protocol stateless for routing
+    nodes.  Discussion remains ongoing as of this writing, with a focus
+    on preventing the mechanism from being abused for spam and for
+    enhancing privacy.
+
+FIXME:jnwebery-sbc-report
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{%
+endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure.
+Please consider upgrading to new releases or helping to test release
+candidates.*
+
+- [Bitcoin Core 0.19.1][] (release candidate)
+
+- [LND 0.9.1][] (release candidate)
+
+FIXME:harding:update-links-tuesday-evening
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #13339][] wallet: Replace %w by wallet name in -walletnotify script FIXME:bitschmidty
+
+- [Eclair #1325][] allows the SendToRoute endpoint to accept routing
+  hints that can help the spending node find a path to the receiving
+  node.
+
+- [BIPs #886][] updates [BIP340][] schnorr signatures with two
+  recommendations: (1) to include entropy in the nonce calculation
+  whenever a random number generator is available, and (2) to verify any
+  signatures produced by the software are valid before distributing them
+  to outside programs, at least whenever the extra computation and delay
+  isn't burdensome.  These two steps help ensure an attacker can't
+  recover a user's private key by obtaining invalid signatures with
+  reused nonces; see [Newsletter #83][news83 nonce safety] for attack
+  details.  Other similar changes to BIP340 are still being [worked
```

Comment:
This PR has now been merged. Should the "still being worked on" wording be updated?